SpEC/com_motion.py

```diff
@@ -208,10 +208,8 @@ def remove_avg_com_motion(path_to_waveform_h5='rhOverM_Asymptotic_GeometricUnits
     # Read the waveform data in
     w_m = read_from_h5(path_to_waveform_h5)
 
-    # Insert '_CoM' into the original `path_to_waveform_h5`; this will be the output path
-    path_to_new_waveform_h5 = path_to_waveform_h5.split('.h5', 1)
-    path_to_new_waveform_h5.insert(1, '_CoM.h5')
-    path_to_new_waveform_h5 = ''.join(path_to_new_waveform_h5).replace(w_m.descriptor_string + '_', '')
+    # Output filename: insert '_CoM' into the original `path_to_waveform_h5`
+    path_to_new_waveform_h5 = path_to_waveform_h5.replace('.h5', '_CoM.h5')
 
```

Comment:
**Action item:** Please add `, 1` to the end of the `replace` argument, and tack on `.replace(w_m.descriptor_string + '_', '')`.